src/tools/miri/tests/fail/dangling_pointers/dangling_pointer_offset.stderr

```diff
@@ -0,0 +1,15 @@
+error: Undefined Behavior: out-of-bounds pointer arithmetic: ALLOC has been freed, so this pointer is dangling
+  --> $DIR/dangling_pointer_offset.rs:LL:CC
+   |
+LL |     let x = unsafe { p.offset(42) };
+   |                      ^^^^^^^^^^^^ out-of-bounds pointer arithmetic: ALLOC has been freed, so this pointer is dangling
```

Comment:
This is the key change, previously this talked about "pointer to alloc859 was dereferenced after this allocation got freed"